src/dlgProfilePreferences.cpp

```diff
@@ -128,6 +130,25 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
     // on top! 8-)
     tabWidget->setCurrentIndex(0);
 
+    // To be moved to a slot that is used on GUI language change when that gets
+    // implimented:
+    pushButton_showGlyphUsage->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
+                                          .arg("<p>This will bring up a display showing all the symbols used in the current "
+                                               "map and whether they can be drawn using just the specifed font, any other "
+                                               "font, or not at all.  It also shows the sequence of Unicode <i>code-points</i> "
+                                               "that make up that symbol, so that they can be identified even if they "
+                                               "cannot be displayed; also, up to the first thirty two rooms that are using "
+                                               "that symbol are listed, which may help to identify any unexpected or odd cases.<p>"));
+    fontComboBox_mapSymbols->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
+                                        .arg("<p>Select the only or the primary font used (depending on <i>Only use symbols "
+                                             "(glyphs) from choosen font</i> setting) to produce the 2D mapper room symbols.</p>"));
+    checkBox_onlyUseChoosenFont->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
```

Comment:
Same.

Reply:
*Sigh* That is a spelling error I keep making - it isn't something I have **cho~o~sen** to do and Qt Creator does not have a smell-checker for variable names. 😜